modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -650,6 +668,15 @@ namespace gapi {
         return pkg;
     };
 
+    template<typename... FF>
+    GKernelPackage kernels(FF&... functors)
```

Comment:
What about `kernels<Ts...>(Fs...)`? Would it look uglier from the user perspective?

Reply:
I think it isn't necessary now